src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/NotSuitableSDOCallbackHandler.java

```diff
@@ -38,10 +40,13 @@ public class NotSuitableSDOCallbackHandler extends CallbackHandler {
 
     private final Time time;
 
+    private static final int lengthAllowed = 150;
```

Comment:
If this is only used in the method move it within the method, i.e not class level

Reply:
it's easier to spot these numbers on class level or even in application.yml. Also, just using the number in the method incurs "magic number" warnings